src/util.js

```diff
@@ -271,7 +271,7 @@ function timeif(bool, name, thunk) {
 }
 
 function pipeline(fns) {
-  return _.flowRight.apply(null, fns.reverse());
+  return _.flow.apply(null, fns);
```

Comment:
I think `_.flow(fns)` would be equivalent for our purposes, since `_.flow` can take arrays as well. But at that point, it's not clear that we even need the `pipeline` function, since `_.flow` is fewer characters than `util.pipeline`.